lib/utils/transcription-utils.js

```diff
@@ -856,7 +856,7 @@ module.exports = (logger) => {
           {VERBIO_SPEECH_INCOMPLETE_TIMEOUT: verbioOptions.speech_incomplete_timeout}),
       };
     } else if (vendor.startsWith('custom:')) {
-      let {options = {}} = rOpts;
+      let {options = {}} = rOpts.customOptions || {};
```

Comment:
shouldnt this simply be
```js
let options = rOpt.customOptions || {};
```

Reply:
inside customOptions, there is options attribute. and we want to use that options.